config/cmake/H5pubconf.h.in

```diff
@@ -222,6 +222,9 @@
 /* Define to 1 if you have the `dmalloc' library (-ldmalloc). */
 #cmakedefine H5_HAVE_LIBDMALLOC @H5_HAVE_LIBDMALLOC@
 
+/* Define to 1 if you have the <libgen.h> header file. */
+#cmakedefine H5_HAVE_LIBGEN_H @H5_HAVE_LIBGEN_H@
+
```

Comment:
libgen.h is used for basename and dirname, which are only used in the tools. This stuff may go away due to differing semantics between them and their Win32 equivalents.